packages/cli/src/controller/migrate/constants.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import {NETWORK_FAMILY} from '@subql/common';
+import {convertEthereumDs, convertEthereumTemplate} from './manifest/ethereum';
+import {DsConvertFunction, TemplateConvertFunction, NetworkExampleProject} from './types';
+
+// @ts-ignore
+export const networkDsConverters: Record<NETWORK_FAMILY, DsConvertFunction> = {
```

Comment:
I think its worth combining all these `*Converters` into one record, with the keys being a complete object. 

That way when implementing a new network family there's enforcement that everything is implemented and its easier to throw an error if a network is unsupported